src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl

```diff
@@ -111,6 +111,34 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="note">
+    <xsl:variable name="type">
+      <xsl:value-of select="normalize-space(@type)"/>
+    </xsl:variable>
+
+    <div>
+      <xsl:attribute name="class">
+        <xsl:choose>
+          <xsl:when test="$type = 'warning'">WARNING alert alert-warning</xsl:when>
+          <xsl:when test="$type = 'caution'">CAUTION alert alert-danger</xsl:when>
+          <xsl:when test="$type = 'important'">IMPORTANT alert alert-danger</xsl:when>
+          <xsl:otherwise>NOTE alert alert-info</xsl:otherwise>
+        </xsl:choose>    
+      </xsl:attribute>
+      <h5>
```

Comment:
@richardschneider, the default behavior of DFM would transform to `<h5>NOTE</h5>`, but as @vwxyzh pointed, user can configure to use , for example, `<h6>NOTE</h6>` through `token.json`. Could the transformation of the xml comments also follow this?